frontend/src/pages/mainPage/components/PastBattleCard.tsx

```diff
@@ -0,0 +1,96 @@
+import { Link } from 'react-router-dom';
+import { type BattleCardItem } from '../types/battle';
+import PeopleIcon from '@/assets/icon/people.svg?react';
+import ClockIcon from '@/assets/icon/clock.svg?react';
+import TrophyIcon from '@/assets/icon/trophy.svg?react';
+
+import { TEAM_STYLE } from '../types/team';
+import IconBox from './IconBox';
+
+export default function PastBattleCard({ item }: { item: BattleCardItem }) {
+  const teamStyle = TEAM_STYLE[item.winner ?? 'A'];
+
+  return (
+    <div className="w-full rounded-2xl bg-[#1A1A2E] overflow-hidden">
+      {/* 상단 바 */}
+      <div className="h-1 w-full" style={{ backgroundColor: teamStyle.color }} />
```

Comment:
주로 정적인 부분은 tailwindcss에서 런타임간 동적으로 스타일이 바뀌는 부분은 어쩔 수 없이 style을 사용하는데요 . 
해당 영역은 런타임간에 동적으로 컬러가 바뀌지 않는 부분으로 추정되는데.. 그런 경우 suggestion 처럼 적용 하셔도 될 것 같습니다. 

```suggestion
<div className="h-1 w-full bg-[#teamStyle.color]"/> 
```

Reply:
아하 몰랐었는데 덕분에 배워갑니다! 수정했습니다!!